src/python/genny/cedar_report.py

```diff
@@ -94,12 +94,11 @@ def created_at(self):
 
 def build_report(config):
     artifacts = []
-
     for path in config.metrics_file_names:
         base_name = os.path.basename(path)
         a = CedarTestArtifact(
             bucket=config.cloud_bucket,
-            path=base_name,
+            path='metrics/{}'.format(base_name),
```

Comment:
request - can you put a bit more pathing here e.g. `metrics/${format}/yyyy/`? I've learned that grouping artifacts at least by year really helps with browsing the bucket and easily purging stale data if/when required.

Reply:
Noted. It turns out cedar will automatically prepend the task ID to the path. The task ID is unique for each execution. So I don't anything here more than the base name